src/plugins/sentry.ts

```diff
@@ -0,0 +1,21 @@
+
+import { AWSLambda } from '@sentry/serverless';
+
+const SENTRY_DSN = process.env.SENTRY_DSN;
+
+export function initSentry() {
+  if (!SENTRY_DSN) return;
+
+  AWSLambda.init({
+    dsn: SENTRY_DSN,
+  
+    tracesSampleRate: 1.0,
+  });
+}
+
+export function wrapHandler(handler) {
+  if (!SENTRY_DSN) return handler;
+
+  initSentry();
```

Comment:
I am not sure here, but due to the logic of the code we will invoke `initSentry` function every time when we, for example, update pools?
shouldn't it be initialised once when server is started?

Reply:
This runs in a lambda, which is a fresh start every time. It only runs once in the main lambda wrapper.